IM_calculation/IM/im_calculation.py

```diff
@@ -2,19 +2,19 @@
 import glob
 import os
 import sys
+import shutil
 from datetime import datetime
 from functools import partial
-from multiprocessing.pool import Pool
-from collections import ChainMap
+from pathlib import Path
 from typing import List, Iterable
 
+from mpi4py import MPI
```

Comment:
no name main wrapper for this? Makes it impossible to import any of the individual IM functions without MPI running in the background